### DISCUSSION TITLE
Error in user guide Mars retrograde script

### DISCUSSION THREAD

**Discussion by rolomoto** [Stellarium/stellarium#1936] [Q&A]
I am trying to run the script in section 16.5 of the manual but when I add the following after the header I get an error:
Parse error at line 12. Line 12 appears to start with var apx = Label.....

var color = "#ff9900";
var info = LabelMgr.labelScreen("A motion of Mars", 20, 20,
           false, 24, color);
var apx = LabelMgr.labelScreen("Setup best viewing angle, FOV
          and date/time.", 20, 50, false, 18, color);
LabelMgr.setLabelShow(info, true);
LabelMgr.setLabelShow(apx, true);
core.wait(2);
LabelMgr.setLabelShow(apx, false);

**Comment by alex-w** [comment]
Hint:
```
var color = "#ff9900";
var info = LabelMgr.labelScreen("A motion of Mars", 20, 20, false, 24, color);
var apx = LabelMgr.labelScreen("Setup best viewing angle, FOV and date/time.", 20, 50, false, 18, color);
LabelMgr.setLabelShow(info, true);
LabelMgr.setLabelShow(apx, true);
core.wait(2);
LabelMgr.setLabelShow(apx, false);
```

**Reply by rolomoto** [reply]
Thanks Alex! Can I ask, is it possible to get back to normal view (out of the script) without quitting the program and restarting?

**Reply by alex-w** [reply]
Please use Script Console (F12)
